cmd/sgx_plugin/README.md

```diff
@@ -63,11 +64,36 @@ The component has the same basic dependancies as the
 [generic plugin framework dependencies](../../README.md#about).
 
 The SGX device plugin requires Linux Kernel SGX drivers to be available. These drivers
-are currently available via [RFC patches on Linux Kernel Mailing List](https://git.kernel.org/pub/scm/linux/kernel/git/jarkko/linux-sgx.git/tag/?h=v39).
-RFC *v39* was used to validate what is written in this document.
+are currently available via [RFC patches on Linux Kernel Mailing List](https://git.kernel.org/pub/scm/linux/kernel/git/jarkko/linux-sgx.git).
+RFC *v41* was used to validate what is written in this document.
 
 The hardware platform must support SGX Flexible Launch Control.
 
+#### Backwards compatibility note
+
+The SGX device nodes have changed from /dev/sgx/[enclave|provision]
+to /dev/sgx_[enclave|provision] in v4x RFC patches according to the
+LKML feedback.
+
+Backwards compatibility is provided by adding /dev/sgx directory mount
+to containers. This assumes the cluster admin has installed the udev
+rules provided below to make the old device nodes as symlinks to the
+new device nodes.
+
+```bash
+$ cat /etc/udev/rules/9*.rules
+SUBSYSTEM=="misc",KERNEL=="enclave",MODE="0666"
+SUBSYSTEM=="misc",KERNEL=="sgx_enclave",MODE="0666",SYMLINK+="sgx/enclave"
+SUBSYSTEM=="sgx",KERNEL=="sgx/enclave",MODE="0666"
+SUBSYSTEM=="misc",KERNEL=="provision",GROUP="sgx_prv",MODE="0660"
+SUBSYSTEM=="misc",KERNEL=="sgx_provision",SYMLINK+="sgx/provision",GROUP="sgx_prv",MODE="0660"
```

Comment:
Why different permissions for `sgx/enclave` and `sgx/provision`? Doesn't the `o` bit affect what the container workload can do, if it runs as a regular user? Also, I think symbolic links don't have a mode as such; they only can be changed by a user who has write access to the underlying directory.

Reply:
@ipuustin no particular reason. I was using the rules mainly targeted for bare metal usage. I dropped the `GROUP` settings.